assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -162,7 +162,7 @@ che.oauth.github.tokenuri= https://github.com/login/oauth/access_token
 che.oauth.github.redirecturis= http://localhost:${CHE_PORT}/api/oauth/callback
 
 # You can setup GitHub OAuth to automate authentication to remote repositories.
-# You need to first register this application with GitHub OAuth.
+# You need to first register this application with OpenShift OAuth.
```

Comment:
>You need to first register this application with OpenShift OAuth.

Are you sure this is a good idea?

Reply:
Removed the leftovers

Reply:
I believe we need some explanation of what is it and how to get it.

Reply:
Updated the description